torchao/custom_ops.py

```diff
@@ -0,0 +1,8 @@
+import torch
+from torch import Tensor
+
+def nms(boxes: Tensor, scores: Tensor, iou_threshold: float) -> Tensor:
+    """
+    See https://pytorch.org/vision/main/generated/torchvision.ops.nms.html
+    """
+    return torch.ops.torchao.nms.default(boxes, scores, iou_threshold)
```

Comment:
My understanding was this path should just work with torch.compile, should we test that explicitly?

Reply:
Yup